src/main/java/com/dislinkt/post/service/PostService.java

```diff
@@ -39,6 +39,12 @@ public List<PostDTO> findByPersonId(UUID userId, UUID personId) throws Exception
         boolean isOwner = userId.compareTo(personId) == 0;
         Person user = personService.findOne(userId);
         Person person = personService.findOne(personId);
+
+        if (user.getBlockedBy().contains(person))
+            throw new Exception("This user has you blocked!");
+        if (person.getBlockedBy().contains(user))
+            throw new Exception("You have blocked this user!");
```

Comment:
I would consider using different naming for user and person, since both are persons, but one is a current user.

Reply:
Thanks for the tip: I will see to it in the branch I am currently working on (since it is just a small name change).